crates/burn-import/src/onnx/op_configuration.rs

```diff
@@ -745,6 +745,49 @@ pub fn layer_norm_config(node: &Node) -> (LayerNormConfig, bool) {
     )
 }
 
+/// Create a PadConfig from the attributes of the node
+pub fn pad_config(node: &Node) -> PadConfig {
+    fn get_pads(node: &Node) -> Vec<usize> {
+        let input_dim = match &node.inputs.first().unwrap().ty {
+            ArgType::Tensor(tensor) => tensor.dim,
+            _ => panic!("Only tensor input is valid"),
+        };
+        let pads: Vec<usize> = match &node.inputs[1].value {
+            Some(Data::Int64s(shape)) => shape
+                .iter()
+                .map(|&x| {
+                    if x < 0 {
+                        // TODO: support negative pads
+                        panic!("Negative pad is not supported");
+                    }
+                    x as usize
+                })
+                .collect(),
+            _ => panic!("Pads data type must be int64"),
+        };
+
+        assert_eq!(pads.len(), input_dim * 2);
+
+        if input_dim == 1 {
+            vec![pads[0], pads[1], 0, 0]
+        } else {
+            let left = pads[input_dim - 1];
+            let top = pads[input_dim - 2];
+            let right = pads[pads.len() - 1];
+            let bottom = pads[pads.len() - 2];
+            vec![left, right, top, bottom]
```

Comment:
This assumes the user wants to pad last 2 dimensions which works for the Burn's pad but it's silent. We should break if ONNX configuration does not fit Burn's API. Obviously ONNX version is more generic, which requires Burn's code be updated as well (not required for this PR).